mongo/client_test.go

```diff
@@ -179,18 +179,59 @@ func TestClient(t *testing.T) {
 		assert.Equal(t, rc, client.readConcern, "expected read concern %v, got %v", rc, client.readConcern)
 	})
 	t.Run("retry writes", func(t *testing.T) {
+		testURI := "mongodb://localhost:27017/?retryWrites=false"
```

Comment:
Nit: it's more confusing to have these as variables, since it looks like they should apply to a larger scope than they actually do.  Consider inlining them.

Reply:
inlining them makes the test struct definitions kind of long and hard to read so I renamed these `retryWritesURI` and `retryWritesErrorURI` (same for the retryable reads URIs).

Reply:
Still would prefer inlining, but let's see if Isabella will break the tie.

Reply:
I'm fine with them separate

Reply:
@jyemin I agree that inlining would make sense as the variables are only used once, but I've been trying to avoid long lines in the code base because they don't fit well on screens and Github and making the structs multi-lined would decrease readability.

Reply:
So long as we agree on the general principle of limiting the scope of variables, I'm fine with exceptions where it decreases readability.